mbrl/env/ant_truncated_obs.py

```diff
@@ -16,18 +29,23 @@ def step(self, a):
         forward_reward = (xposafter - xposbefore) / self.dt
         ctrl_cost = 0.5 * np.square(a).sum()
         contact_cost = (
-            0.5 * 1e-3 * np.sum(np.square(np.clip(self.sim.data.cfrc_ext, -1, 1)))
+            0.5 * 1e-3 * np.sum(np.square(np.clip(self.data.cfrc_ext, -1, 1)))
```

Comment:
Do any of the changes like this change the dynamics? The `.sim` objects were always mysterious.

Reply:
I think that's just the API change between the `mujoco_py` vs `mujoco` migration, actually (Most of the math remains the same)